envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/SnapshotProperties.kt

```diff
@@ -157,6 +157,7 @@ class CanaryProperties {
 
 class TrafficSplittingProperties {
     var zoneName = ""
+    var headerName = ""
```

Comment:
you can add some default value here

Reply:
I have this empty value just to have an option not to add this header, and it can be configured as default in application.yaml for example?